src/components/SummaryPage.tsx

```diff
@@ -7,50 +8,196 @@ import { Notes } from './summaryPage/Notes.tsx';
+    results[FILES.PHENOTYPE] = dataToPull.Latest[
+      FILES.PHENOTYPE
+    ].results.bindings.map((item: any) => {
+      let filteredItem = null;
+      if (dataToPull.Previous[FILES.PHENOTYPE]) {
+        // @ts-expect-error Explanation: Handling the data properly
+        filteredItem = dataToPull.Previous[
+          FILES.PHENOTYPE
+        ].results.bindings.filter(
+          (prevItem: any) =>
+            prevItem?.phenotype?.value === item?.phenotype?.value,
+        );
+      }
+      if (filteredItem.length) {
+        return {
+          label: item?.phenotype?.value,
+          count: item?.count?.value,
+          change: item.count.value - filteredItem[0].count.value,
+        };
+      } else {
+        return {
+          label: item?.phenotype?.value,
+          count: item?.count?.value,
+          change: 0,
+        };
+      }
+    });
+
+    // @ts-expect-error Explanation: Handling the data properly
+    results[FILES.POPULATION] = dataToPull.Latest[
+      FILES.POPULATION
+    ].results.bindings.map((item: any) => {
+      let filteredItem = null;
+      if (dataToPull.Previous[FILES.POPULATION]) {
+        // @ts-expect-error Explanation: Handling the data properly
+        filteredItem = dataToPull.Previous[
+          FILES.POPULATION
+        ].results.bindings.filter(
+          (prevItem: any) => prevItem?.model?.value === item?.model?.value,
+        );
+      }
+      if (filteredItem.length) {
+        return {
+          label:
+            item?.model?.value + '  (' + item?.neuron_category?.value + ')',
+          count: item?.count?.value,
+          change: item.count.value - filteredItem[0].count.value,
+        };
+      } else {
+        return {
+          label:
+            item?.model?.value + '  (' + item?.neuron_category?.value + ')',
+          count: item?.count?.value,
+          change: 0,
+        };
+      }
+    });
+
+    // @ts-expect-error Explanation: Handling the data properly
+    results[FILES.SPECIES] = dataToPull.Latest[
+      FILES.SPECIES
+    ].results.bindings.map((item: any) => {
+      let filteredItem = null;
+      if (dataToPull.Previous[FILES.SPECIES]) {
+        // @ts-expect-error Explanation: Handling the data properly
+        filteredItem = dataToPull.Previous[
+          FILES.SPECIES
+        ].results.bindings.filter(
+          (prevItem: any) => prevItem?.type?.value === item?.type?.value,
+        );
+      }
+      if (filteredItem.length) {
+        return {
+          label: item?.type?.value + '  (' + item?.phenotype_label?.value + ')',
+          count: item?.count?.value,
+          change: item.count.value - filteredItem[0].count.value,
+        };
+      } else {
+        return {
+          label: item?.type?.value + '  (' + item?.phenotype_label?.value + ')',
+          count: item?.count?.value,
+          change: 0,
+        };
+      }
+    });
+
+    setLoaded(true);
+    setData(results);
   }, []);
 
-  if (!data || !labels)
+  if (!loaded)
```

Comment:
Note (non-blocking): My understanding is that the loader will never show given how we are fetching the data